client/gutenberg/extensions/map/save.js

```diff
@@ -10,11 +10,11 @@ class MapSave extends Component {
 	render() {
 		const { className, attributes } = this.props;
 		const { mapStyle, mapDetails, points, zoom, mapCenter, markerColor } = attributes;
-		const pointsList = points.map( point => {
+		const pointsList = points.map( ( point, index ) => {
 			const { longitude, latitude } = point.coordinates;
-			const url = 'https://www.google.com/maps/search/?api=1&&query=' + latitude + ',' + longitude;
+			const url = 'https://www.google.com/maps/search/?api=1&query=' + latitude + ',' + longitude;
 			return (
-				<li>
+				<li key={ index }>
```

Comment:
List index is generally discouraged as a key, stable identifiers are preferred ([#](https://reactjs.org/docs/lists-and-keys.html#keys)). We seem to have `point.id`, can we use that?

https://github.com/Automattic/wp-calypso/blob/4758f738cdb18410db3516e732e561979770a187/client/gutenberg/extensions/map/edit.js#L57